ExampleApp.xcodeproj/xcshareddata/xcschemes/ExampleApp.xcscheme

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@vault087 can you share more about why we are committing this new `.xcscheme` file?

Reply:
That's Xcode internal files, not really sure why it created this file just now.